src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java

```diff
@@ -182,5 +208,14 @@ private void installPlugins(List<PluginSpec> install) {
                 }
             };
         }
+
+        private void handleInvalidState(String pluginEvaluationOutcome, PluginSpec spec, Description d) {
+            final String format = String.format("%s plugin is required by test %s but it is not installed in a valid version and ATH is running in preconfigured mode", spec, d.getDisplayName());
+            if (pluginEvaluationOutcome.equals(FAIL_ON_INVALID)) {
+                throw new IllegalStateException(format);
```

Comment:
`AssertionError` is the normal throwable type for test failures.

Reply:
Fixed